include/glow/Graph/Graph.h

```diff
@@ -679,6 +690,15 @@ class Function final : public Named {
 
   /// \returns pointer to the class member for the nodes list.
   static NodesList Function::*getNodesMemberPtr() { return &Function::nodes_; }
+
+  /// Replace with \p newTy, the type of all the nodes, placeholders and so on
+  /// that use \p origTy.
+  /// If a \p context is provided, this method makes sure the tensors
+  /// backing the placeholders are properly updated. Otherwise,
+  /// this method assumes the placeholders are not bound to anything.
+  /// \pre The parent module has only one function, this one.
+  void mutateNodesType(ElemKind origTy, ElemKind newTy,
```

Comment:
Why is this a member of the module? The cloner and the quantizer are methods that operate outside the Module, not as members. This methods does not need to know the internals of the Module.

Reply:
Yes and no. We need to know what structures need to be updated. It happens that all what need to be updated in visible from the outside (node, placeholder and variable) but it could’ve not been the case. Moving it elsewhere exposes us to tighten the coupling between the internals of the graph and whatever does the conversion.

Reply:
I don't understand this part. The optimizer is not a part of the graph. Right? 

You don't need to be a part of the internals of Function to be able to mutate it.